package.json

```diff
@@ -23,9 +23,11 @@
     "node": ">=18.0"
   },
   "devDependencies": {
+    "@redocly/cli": "^1.22.1",
```

Comment:
Oooo love it -- as an aside, in another project I am using this: https://www.npmjs.com/package/swagger-spec-validator but for a few reasons have been wanting to find an alternative.  🎉